BookingsApi/BookingsApi.UnitTests/DAL/Services/FeatureFlagServiceTests.cs

```diff
@@ -64,14 +68,15 @@ public void GetFeatureFlag_Should_Return_True_for_EJud_Feature()
         [Test]
         public void GetFeatureFlag_Should_Return_False_for_EJud_Feature()
         {
+            _mocker.Mock<IFeatureToggles>().Setup(opt => opt.EJudFeature()).Returns(false);
             _mocker.Mock<IOptions<FeatureFlagConfiguration>>().Setup(opt => opt.Value).Returns(new FeatureFlagConfiguration()
             {
-                EJudFeature = false
```

Comment:
Same as above, test describes Ejud

Reply:
just added it in case would fail to initialise the service

Reply:
removed